contributors/working_with_datastructures.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: documentation
+title:  Working with Datastructures
+order: 2
+---
+
+## 
+
+Check out the example on how to work with a table [here](https://github.com/Caleydo/lineage/blob/master/src/dataExplorations.ts). 
```

Comment:
The plan is to create a new demo repository, but it's here now, and this is better than not having it.